resource_customizations/apps/Deployment/actions/action_test.yaml

```
@@ -7,4 +7,9 @@ actionTests:
   expectedOutputPath: testdata/deployment-pause.yaml
 - action: resume
   inputPath: testdata/deployment-pause.yaml
-  expectedOutputPath: testdata/deployment-resume.yaml
\ No newline at end of file
+  expectedOutputPath: testdata/deployment-resume.yaml
+- action: scale
+  inputPath: testdata/deployment.yaml
+  expectedOutputPath: testdata/deployment-scaled.yaml
+  parameters:
+    scale: "6"
```

```suggestion
    scale: "6"
```
Don't add missing newline at eof

---

Right, that's effectively a warning. Don't do that. Add a newline.

---

Thank you! should I add this as a comment in the top?

---

No, just add an extra newline at the end of each file

---

Added